startup.m

```diff
@@ -0,0 +1,6 @@
+% startup.m
+setPath ;
+setenv('DJ_HOST', 'at-database.ad.bcm.edu');
+setenv('DJ_USER','spatel');
+setenv('DJ_PASS','spatel');
```

Comment:
argh!  checked in password.  Please change your password and exclude your `startup.m` from the path or set the environment variables in a separate script such as your `~/.profile` (on Mac OS)